src/vs/base/parts/quickinput/browser/quickInputList.ts

```diff
@@ -236,6 +234,7 @@ class ListElementRenderer implements IListRenderer<ListElement, IListElementTemp
 
 	disposeElement(element: ListElement, index: number, data: IListElementTemplateData): void {
 		data.toDisposeElement = dispose(data.toDisposeElement);
+		data.actionBar.clear();
```

Comment:
What happens when you launch a pick pick with Actions and then before disposable you launch a different quick pick? Like `cmd+shift+p` into `cmd+shift+o` or one where the 2nd doesn't have buttons... like `cmd+shift+p` into `> Configure Display Language`?

Reply:
Good idea. I confirmed that in this case we first call `disposeElement` and then `renderElement`. Here's going to configure display language:

![Screen Shot 2022-10-14 at 10 36 49 AM](https://user-images.githubusercontent.com/12821956/195907656-7f44653c-8f3f-480b-b8f1-b9f646fe6435.png)


The `renderElement` calls at the top are for the command palette items. Then we dispose of them all and call `renderElement` for the 3 new items that show in the display language ui